### PR TITLE
makes dionae the supreme race like we always wanted!

### PR DESCRIPTION
**Pull request by diegoxt45**
thanks pomf.

### CODE REVIEW

File: code/modules/mob/living/carbon/species.dm [hunk 1]
```diff
@@ -968,15 +968,12 @@ var/global/list/whitelisted_species = list("Human")
 	flesh_color = "#907E4A"
 
 	has_mutant_race = 0
-	burn_mod = 2.5 //treeeeees
+	burn_mod = -100 //FUCK YOU DIONA REIGNS SUPREME
```
Comment (boy2mantwicethefam) [inline code/modules/mob/living/carbon/species.dm]: wat

File: code/modules/mob/living/carbon/species.dm [hunk 2]
```diff
@@ -968,15 +968,12 @@ var/global/list/whitelisted_species = list("Human")
...
 
-	move_speed_mod = 7
+	
 
-	species_intro = "You are a Diona.<br>\
-					You are a plant, so light is incredibly helpful for you, in both photosynthesis, and regenerating damage you have received.<br>\
-					You absorb radiation which helps you in a similar way to sunlight. Your rigid, wooden limbs make you incredibly slow.<br>\
-					You do not need to breathe, do not feel pain,  you are incredibly resistant to cold and low pressure, and have no blood to bleed.<br>\
-					However, as you are a plant, you are incredibly susceptible to burn damage, which is something you can not regenerate normally."
+	species_intro = "You are the biggest chad ever."
```
Comment (boy2mantwicethefam) [inline code/modules/mob/living/carbon/species.dm:976]: based

File: code/modules/mob/living/carbon/species.dm [hunk 3]
```diff
@@ -1175,7 +1172,9 @@ var/list/has_died_as_golem = list()
 
 	gender = NEUTER
 
-	tox_mod = 2
+	tox_mod = 9.9
+    burn_mod = 9.9
+    brute_mod = 9.9
```
Comment (boy2mantwicethefam) [inline code/modules/mob/living/carbon/species.dm]: I think these have bad indent

File: code/modules/mob/living/carbon/species.dm [hunk 4]
```diff
@@ -1293,8 +1292,9 @@ var/list/has_died_as_golem = list()
 	anatomy_flags = HAS_LIPS | HAS_SWEAT_GLANDS
 
 	default_mutations=list(RAD_IMMUNE)
-	burn_mod = 1.1
-	tox_mod = 0.5
+	burn_mod = 9.9
+	tox_mod = 9.9
+    brute_mod = 9.9
```
Comment (boy2mantwicethefam) [inline code/modules/mob/living/carbon/species.dm]: same here

**Comment by PJB3005** [comment]
Speed merge and make this man a collaborator.

**Comment by boy2mantwicethefam** [comment]
Speed merge and make this man a headmin.

File: code/modules/mob/living/carbon/species.dm (hunk 3 above)
Comment (diegoxt45) [reply code/modules/mob/living/carbon/species.dm]: nah man certified legit game balance

File: code/modules/mob/living/carbon/species.dm (hunk 1 above)
Comment (boy2mantwicethefam) [reply code/modules/mob/living/carbon/species.dm]: by the way make this a 0 if you want invincibility

**Comment by ShiftyRail** [comment]
Due to a call for strike by several representative trade unions of /vg/station, we are not able to perform code review for this PR.
We apologise for the inconvience.

File: code/modules/mob/living/carbon/species.dm (hunk 3 above)
Comment (boy2mantwicethefam) [reply code/modules/mob/living/carbon/species.dm]: Remember to use Tab for indenting (making the space beneath the code), not space bar

File: code/modules/mob/living/carbon/species.dm [hunk 5]
```diff
@@ -946,8 +946,8 @@ var/global/list/whitelisted_species = list("Human")
 	deform = 'icons/mob/human_races/r_def_plant.dmi'
 	known_languages = list(LANGUAGE_ROOTSPEAK)
 	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat/diona
-	attack_verb = "slashes"
-	punch_damage = 5
+	attack_verb = "OBLITERATES"
+	punch_damage = 1000
```
Comment (boy2mantwicethefam) [inline code/modules/mob/living/carbon/species.dm:950]: image

**Comment by Exxion** [comment]
speedmerge

**Comment by ShiftyRail** [comment]
lol

**Comment by boy2mantwicethefam** [comment]
based

**Comment by Exxion** [comment]
Classic

**Comment by SonixApache** [comment]
what

**Comment by ProfanedBane** [comment]
The hero /vg/station needed.

**Pull request merged**